src/NUnitEngine/nunit.engine.tests/nunit.engine.tests.csproj

```diff
@@ -42,12 +42,6 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Mono.Cecil.0.9.6.1\lib\net20\Mono.Cecil.dll</HintPath>
     </Reference>
-    <Reference Include="Mono.Cecil.Mdb">
```

Comment:
I guess we need separate package files for Linux then. I was used to the old NAnt scripts, where you just do everything using a forward slash. Apparently not so with MsBuild.